tests/Unit/DataStructures/Tensor/Expressions/Test_TensorExpressions.cpp

```diff
@@ -248,6 +257,20 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Tensor.Expression.Evaluate",
       ti_F, ti_A, ti_C, ti_D>();
 }
 
+SPECTRE_TEST_CASE(
+    "Unit.DataStructures.Tensor.Expression.ContainsIndicesToContract",
+    "[DataStructures][Unit]") {
+  test_contains_indices_to_contract<ti_a, ti_b, ti_c>(false);
```

Comment:
Could you put these checks in the `Unit.DataStructures.Tensor.Expression.Evaluate` test case wrapped in a function, eg
```cpp
void test_contains_indices_to_contract() {
  INFO("Checking contains_indices_to_contract");
  ...
}
```
The reason being that each test case, especially small ones like this, have more overhead to launch than actual runtime, and so reduces the total number of test cases helps reduce test runtime. The quicker all the tests run, the more likely people are to run the tests :)

I'd suggest making your life easier (fewer conflicts) and just doing this "combine test cases" in the second commit in this PR :)

Reply:
Done!